gradle-scripts/git-publish.gradle

```diff
@@ -0,0 +1,67 @@
+apply plugin: 'org.ajoberstar.git-publish'
+
+gitPublish {
+    // NOTE:
+    // 1) $GRGIT_USER environment variable or "org.ajoberstar.grgit.auth.username" system property must be set
+    // to actual github token, see https://github.com/ajoberstar/gradle-git-publish#migrating-from-orgajoberstargithub-pages
+    // http://ajoberstar.org/grgit/index.html
+    // Also, see gitPublishReset#doFirst below for the implemented fallback for these values.
+    // 2) to use GH token `repoUri` must have an https (not git or ssh) protocol
+
+    repoUri = 'https://github.com/commercetools/commercetools-payone-integration.git'
+    branch = 'gh-pages'
+
+    // what to publish, this is a standard CopySpec
+    contents {
+        into '.'
+        from 'ghpages/' // 1. static content - root index.html page
+
+        with project.copySpec { // 2. test results: both Concordion and logging console outputs
+            into 'latest/'
+            from 'functionaltests/build/reports/'
+        }
+    }
+
+    // message used when committing changes
+    commitMessage = "Publishing tests results (adding commit info here would be cool)"
```

Comment:
`(adding commit info here would be cool)` :) Not meant to be left here I guess

Reply:
it is exactly left by purpose: kind of todo for future.